src/pages/view-test-result/cat-a-mod1/view-test-result.cat-a-mod1.page.ts

```diff
@@ -160,10 +158,27 @@ export class ViewTestResultCatAMod1Page extends BasePageComponent implements OnI
       candidateName: getCandidateName(this.testResult.journalData.candidate),
       candidateDriverNumber: this.testResult.journalData.candidate.driverNumber,
       activityCode: this.testResult.activityCode,
-      testOutcome: getTestOutcomeText(this.testResult),
+      testOutcome: this.getTestOutcomeTextAM1(this.testResult),
```

Comment:
can we use the common getTestOutcomeText here?

Reply:
removed the local function for this and cast as the common object - seems to work :D